### ISSUE TITLE
`(in-hole E hole)` typesets as just `E`

### ISSUE THREAD

**Issue by florence** [racket/redex#210]
In the program:

```
#lang racket
(require redex/reduction-semantics
         redex/pict)
(define-language L
  (E ::= hole (add1 E)))
(render-term L (in-hole E hole))
```

I get back a pict containing just `E`, rather than `E[[]]`, which I would expect. 

https://github.com/samth/plt-history/commit/341d0c76a9bcacebcc959eb8982cf0b5d472f42c#diff-40d2b8fe4358be37c0192b0c40eda357R71

makes this seem like a deliberate choice. However I find myself needing wanting typeset `E[[]]=E` in an explanation, which I currently can't do.

**Comment by rfindler** [comment]
I think that was probably a bad choice on my part and am reluctantly okay
with a backwards incompatible change here.

**Comment by rfindler** [comment]
merged #211.
